### DISCUSSION TITLE
RegisterForRegionNavigation sample, pls

### DISCUSSION THREAD

**Discussion by FeedLab** [PrismLibrary/Prism#2278] [Xamarin.Forms] [answered]
Can someone show me a working sample of how to work with regions in xamarin forms? I got an exception and can't figure out the problem. Using Nuget package 8.0.0.1909

![xamarin](https://user-images.githubusercontent.com/15180718/101983128-27b7d300-3c79-11eb-897e-d98f05b9c7ad.png)

**Comment by dansiegel** [comment, marked as answer]
this is due to a bug with the Unity Container Extension. This is resolved in Prism 8.1 which Sponsors can access on Sponsor Connect. We are considering releasing a service release to address the issue in the new year, otherwise you can just use DryIoc which is a much faster container and works just fine as is.
